zerver/migrations/0311_subscription_is_user_active.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.1.5 on 2021-01-31 15:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    atomic = False
+
+    dependencies = [
+        ('zerver', '0310_jsonfield'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='subscription',
+            name='is_user_active',
+            field=models.BooleanField(),
+        ),
+        migrations.RunSQL("""
+                          UPDATE zerver_subscription
+                          SET is_user_active = zerver_userprofile.is_active
+                          FROM zerver_userprofile
+                          WHERE zerver_subscription.user_profile_id = zerver_userprofile.id
```

Comment:
Do we perhaps want to split this update into batches?

Reply:
We almost certainly do.

Reply:
Let's discuss it on chat, actually.  Even though we're talking millions of rows here, the query is gonna be super easy for postgres to query-plan, so maybe we just let it rip.

Reply:
I would also prefer black-style indentation here.